dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/LoginTests.java

```diff
@@ -81,7 +81,7 @@ public void preconditions()
     }
 
     @Test
-    public void shouldBeAbleToLoginWithOAuth2()
+void shouldBeAbleToLoginWithOAuth2()
```

Comment:
I have the formatting hook setup and it also showed that it is formatting files I staged. I don't understand why this happened in a bunch of files. I am pretty tired of this migration, so unless someone has an automated quickfix I hope we can let this be cleaned up over time 😅

Reply:
I think I solved it.